TestFoundation/TestURL.swift

```diff
@@ -35,7 +35,7 @@ private func getTestData() -> [Any]? {
         XCTFail("Unable to deserialize property list data")
         return nil
     }
-    guard let parsingTests = testRoot[kURLTestParsingTestsKey] as? [Any] else {
+    guard let parsingTests = testRoot?[kURLTestParsingTestsKey] as? [Any] else {
```

Comment:
This is related to [SE-0230](https://github.com/apple/swift-evolution/blob/master/proposals/0230-flatten-optional-try.md).

```swift
guard let testRoot = try? PropertyListSerialization.propertyList(from: data, options: [], format: nil) as? [String : Any] else {
```

This would break builds which uses Swift 5 mode.